test/integration/testdata/config.yaml.tmpl

```diff
@@ -4,13 +4,16 @@ metadata:
   name: istio
 data:
   mesh: |-
-    discoveryAddress: istio-pilot.{{.IstioNamespace}}:8080
+    defaultConfig:
```

Comment:
What is defaultConfig about and why is this needed? Once it is a configMap, it can be overridden.